doc/radosgw/s3-notification-compatibility.rst

```diff
@@ -105,6 +105,12 @@ Event Types
 | ``s3:ReducedRedundancyLostObject``           | Not applicable to Ceph                                      |
 +----------------------------------------------+-----------------+-------------------------------------------+
 
+Behavior of Multipart Upload
```

Comment:
probably just add that as a note:
```
.. note:: 

   In case of multipart upload, an ``ObjectCreated:CompleteMultipartUpload`` notification will be sent at the end of the process
```